vec-redis/setup-redis.sh

```diff
@@ -1,51 +1,47 @@
 #!/usr/bin/env bash
 # Usage: setup-redis.sh [redis-stack-server-options]
+set -eEuo pipefail
```

Comment:
@casperdcl when I was testing adding pipefail was actually causing the last line (the conditional check I've put) to not work properly and the whole script throws error..

Reply:
erm.

```bash
cat <<EOF | bash -
set -eEuo pipefail
((exit 1) || echo works fine)
echo $?
EOF
```

gives:

```
works fine
0
```

on my machine